tests/com_interface.rs

```diff
@@ -1,18 +1,11 @@
-winrt::import!(
-    dependencies
-        os
-    types
-        windows::foundation::{IStringable, IUriRuntimeClass, Uri}
-);
-
 #[test]
 fn com_interface() -> winrt::Result<()> {
     use winrt::AbiTransferable;
     use winrt::ComInterface;
 
-    use windows::foundation::IStringable;
-    use windows::foundation::IUriRuntimeClass;
-    use windows::foundation::Uri;
+    use winrt::IStringable;
```

Comment:
I'm wondering if having the foundation types in a `foundation` module would be better (e.g., `use winrt::foundation::IStringable`

Reply:
Yeah, I'm not sure. I'd prefer to keep `winrt` module types to a minimum but this is more consistent with C# which accounts for the vast majority of existing docs/samples/apps so seems helpful to beginners.

Reply:
Yea that was my though. Having a 1 to 1 mapping is helpful for discoverability.